user_tools/src/spark_rapids_pytools/rapids/tool_ctxt.py

```diff
@@ -45,7 +45,7 @@ def __create_and_set_uuid(self):
 
     def __create_and_set_cache_folder(self):
         # get the cache folder from environment variables or set it to default
-        cache_folder = Utils.get_rapids_tools_env('CACHE_FOLDER', '/tmp/rapids_user_tools_cache')
+        cache_folder = Utils.get_rapids_tools_env('CACHE_FOLDER', '/var/tmp/rapids_user_tools_cache')
```

Comment:
doesn't /var directory require sudo permissions?  just wondering if that will be an issue compared to /tmp top-level dir for cache